docs/docs/mdx/programmatically-creating-pages.md

```diff
@@ -224,19 +224,25 @@ will be rendered as the template for all posts. There's a component,
 `MDXRenderer` which is used by `gatsby-plugin-mdx` that will be used to render any
 programmatically accessed MDX content.
 
+For now, to update imports withing .mdx files, you should re-run your gatsby development environment. Otherwise, it will raise the ReferenceError. To import things dynamically, we will use the `MDXProvider` component and provide all common components we want to use, like `Link`, for example.
```

Comment:
as i understand the MDXProvider docs then you need only supply a link component if you like to replace a default react component?

Reply:
maybe with this rewording?

```suggestion
For now, to update imports withing .mdx files, you should re-run your gatsby development environment. Otherwise, it will raise the ReferenceError. To import things dynamically, we will use the `MDXProvider` component and provide all common components we want to use. In the example it replace the `a` tag with the Gatsby `Link` instead of the default React Link element.
```